src/telemetry/basic_robot_human.rs

```diff
@@ -27,12 +27,15 @@ pub fn increase_counters_from_finished_snippet(
     // Snippet is finished when it stops being valid for correction (user has changed code in a different place) or it timeouts
     fn robot_characters(snip: &SnippetTracker) -> i64 {
         let re = Regex::new(r"\s+").unwrap();
-        (re.replace_all(&snip.grey_text, "").len() as f64 * snip.remaining_percentage) as i64
+        let robot_characters = re.replace_all(&snip.grey_text, "").len() as i64;
+        info!("snippet: {}; robot_characters: {}", snip.grey_text, robot_characters);
+        robot_characters
```

Comment:
If you need this log, introduce yourself: "increase_counters_from_finished_snippet: ........"  Then give more information what these numbers mean.